File: ex_slide2/ex_8_lpi.py
n1 = int(input("enter n1: "))
n2 = int(input("enter n2: "))

if (n1 < n2):
    i = (n1 + 1)
    while (i < n2):
        print(i)
        i += 1
else:
    i = (n2 + 1)
    while (i < n1):
        print(i)
        i += 1

File: ex_slide_3/ex_8_lpi.c
#include<stdio.h>

int main(void)
{
    int n1;
    int n2;

    printf("enter n1: ");
    scanf("%i", &n1);

    printf("enter n2: ");
    scanf("%i", &n2);

    if (n1 < n2)
    {
        for (int i = (n1 + 1); i < n2; i++)
        {
            printf("%i\n", i);
        }
    }
    else
    {
        for (int i = (n2 + 1); i < n1; i++)
        {
            printf("%i\n", i);
        }
    }
}
